README.md

```diff
@@ -22,7 +22,6 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 
 * [Cloud Data Loss Prevention](https://github.com/googleapis/nodejs-dlp) (Beta)
 * [Cloud Firestore](https://github.com/googleapis/nodejs-firestore/) (Beta)
-* [Cloud Pub/Sub](#cloud-pubsub-beta) (Beta)
```

Comment:
I believe this needs a redirect to the new repo, as opposed to being removed.

Reply:
Oops. Agreed, fixed.